fetch/handler.go

```diff
@@ -37,7 +37,7 @@ func newHandler(cdb *datastore.CachedDB, cfg Config, bs *datastore.BlobStore, m
 	}
 }
 
-// handleEpochInfoReq returns the ATXs published in the specified epoch.
+// handleMaliciousIDsReq returns the IDs of all malicious nodes.
```

Comment:
```suggestion
// handleMaliciousIDsReq returns the IDs of all known malicious nodes.
```

Reply:
thanks for the review and clarification!